source/compute/instance-types.rst

```diff
@@ -127,22 +127,29 @@ are inherently not as cost efficient if the accelerator is not used.
     * - c1a-gpu
       - AMD x86-64
       - Milan
-      - Each vCPU is a thread of a pCPU
+      - vCPU is a pCPU (no threading)
       - pCPU are dedicated to a single VM
       - 1 or more NVIDIA RTX A6000 48GB GPU
     * - c2-gpu
       - Intel x86-64
       - Skylake
-      - Each vCPU is a thread of a pCPU
+      - vCPU is a thread of a pCPU
       - pCPU are shared between VMs
-      - Slice of NVIDIA A100 80GB GPU
+      - Slice of an NVIDIA A100 80GB GPU
```

Comment:
```suggestion
      - Slice of a NVIDIA A100 80GB GPU
```

Reply:
No, "an" is correct because NVIDIA begins with a vowel sound (it's pronounced "en-vid-e-a").